pytrello2/board.py

```diff
@@ -48,3 +48,13 @@ def delete_board(self, board_id):
         Deletes a Board object with the given ID.
         """
         return self.client.delete(f"boards/{board_id}")
+
+    def update_board(self, board_id, name, desc):
+        """
+        Updates a Board object
+        """
+        data = {}
+        data["name"] = name
+        data["desc"] = desc
+        board_data = self.client.put("boards/", data)
+        return Board(board_data)
```

Comment:
We started using `**kwargs` for the non-required parameters.
You can check the recent [implementation in `card.py`](https://github.com/pytrello2/pytrello2/blob/34ad7738e4321953f5df494afaa92d22fc485f98/pytrello2/card.py#L39).